tests/integ/test_stored_procedure.py

```diff
@@ -738,12 +740,13 @@ def plus4_then_mod5(session_, x):
         input_types=[IntegerType()],
     )
 
+    with pytest.raises(SnowparkSQLException) as ex_info:
+        plus4_then_mod5_sp(3)
+
     if local_testing_mode:
-        with pytest.raises(ModuleNotFoundError) as ex_info:
-            plus4_then_mod5_sp(3)
+        # Local testing nests the error, but pytest only provides the top level error message
+        assert "Python Interpreter Error" in ex_info.value.message
     else:
-        with pytest.raises(SnowparkSQLException) as ex_info:
-            plus4_then_mod5_sp(3)
```

Comment:
should we keep this part for live?

Reply:
It is still there, just moved above of this if/else block.